examples/functions-as-entry-points/functions_as_entry_points.bal

```diff
@@ -3,8 +3,8 @@ import ballerina/io;
 error unknownOpError = error("unknown operation");
 
 // A function named `main` acts as a default entry point to a Ballerina program.
```

Comment:
Use of "as a" intentional here?

Reply:
Yes.

Reply:
I'm assuming that's because the other default entry point is services? Should we mention that here?

Reply:
Do we need to? We've covered that in the hello-world-service BBE. The reason to specifically mention `main` here is since the rest of the BBE talks about other functions as entry points.

Reply:
You could say "..acts as one of the default.." to make it clearer. I don't think we need to mention services specifically here as that's a bit out of scope.

Reply:
```suggestion
 // A function named `main` acts as one of the default entry points to a Ballerina program.
```